src/Tools/dotnet-watch/test/GlobbingAppTests.cs

```diff
@@ -21,9 +21,10 @@ public GlobbingAppTests(ITestOutputHelper logger)
             _app = new GlobbingApp(logger);
         }
 
-        [Theory]
+        [ConditionalTheory]
         [InlineData(true)]
         [InlineData(false)]
+        [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/23360", Queues = "Debian.9.Arm64;Debian.9.Arm64.Open")]
```

Comment:
Do we know if that name works? The name in msbuild is `(Debian.9.Arm64.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm64v8-a12566d-20190807161036`